packages/babel-plugin-transform-modules-commonjs/src/index.ts

```diff
@@ -211,6 +211,7 @@ export default declare((api, options) => {
                   ? mjsStrictNamespace
                   : strictNamespace,
               noIncompleteNsImportDetection,
+              filename: this.file.opts.filename,
```

Comment:
We also use `rewriteModuleStatementsAndPrepareHeader` in the amd and umd plugins. I don't remember if they all support the `importInterop` option, but if they do they should have the same behavior.

Reply:
Yeah, that's covered by what I already have in the diff, right? Or am I missing something?